test/compilable/ddoc9764.sh

```diff
@@ -1,11 +1,9 @@
 #!/usr/bin/env bash
 
-name=`basename $0 .sh`
-dir=${RESULTS_DIR}/compilable
-output_file=${dir}/${name}.html
+output_file=${RESULTS_DIR}/${TEST_DIR}/${TEST_NAME}.html
```

Comment:
This seems to be quite frequent enough.
```
export OUTPUT_FILE="${RESULTS_DIR}/${TEST_DIR}/${TEST_NAME}"
```

could be part of a next step.

Reply:
this one is common enough, I'll add it to this PR

Reply:
actually, wait, this is specific to this test only...it has the `.html` extension

Reply:
Yes, that's why `export OUTPUT_FILE="${RESULTS_DIR}/${TEST_DIR}/${TEST_NAME}"` which allows the user to pick the extension, but yeah, let's think about this in the next step.

Reply:
Yeah, I wasn't sure the best variables to expose...there's combinations of the source dir vs results dir and different extensions...we can figure it out in the next step like you said